app/(website)/juguete/[slug]/page.tsx

```diff
@@ -11,16 +14,35 @@ interface Props {
 
 async function Page({ params: { slug } }: Props) {
   const toy = await client.fetch<Toy>(toySlugQuery, { slug });
-
+  const images = toy.images;
+  const description = toy.description[0];
```

Comment:
The type of the `description` property shouldn't be an array. I think I mistakenly did it that way, do you mind [changing this](https://github.com/alexxuss1996/juguetear-web/blob/badaac2d2d55ce22c6bdb892c719dbae0cb1058b/types/common.ts#L8)?